cds/modules/deposit/static/js/cds_deposit/avc/components/cdsForm.js

```diff
@@ -267,6 +232,22 @@ function cdsFormCtrl($scope, $http, $q, schemaFormDecorators) {
   this.autocompleteType = function() {
     return that.types.promise;
   }
+
+  /**
+   * Utilities
+   */
+  this.stripCommas = function(string) {
```

Comment:
can you please remove this function from the scope, we don't use it anywhere else :)